release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -815,6 +821,14 @@ down.
 * Private, disconnected cluster installations are not supported for AWS GovCloud when installing in the `us-gov-east-1` region due to a known issue.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881262[*BZ#1881262*]).
 
+* For `oc` commands that require TLS verification, if the certificates do not set a Subject Alternative Name, verification does not fall back to the Common Name field and the command fails with the following error:
```

Comment:
@soltysh @sallyom Can you review this known issue/workaround?

Preview: https://bz-1886892--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-known-issues

Reply:
lgtm

Reply:
lgtm

Reply:
@bergerhoffer I've filed the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1889204 to track this issue.

Reply:
Thanks @zhouying7780! I've updated to add this BZ link.

Does this update look good to you now?

Reply:
/lgtm